prov/efa/src/efa_av.c

```diff
@@ -76,15 +99,15 @@ struct efa_conn *efa_av_addr_to_conn(struct efa_av *av, fi_addr_t fi_addr)
  */
 fi_addr_t efa_av_reverse_lookup(struct efa_av *av, uint16_t ahn, uint16_t qpn)
```

Comment:
This is already the function used for dgram code path. rdm code path uses `efa_av_reserve_lookup_rdm`.